### PR TITLE
Fix/fix feedback p2p buy

### PR DESCRIPTION
**Pull request by vietdangquoc**
# Ticket
#856 

# General
- [ ] No unnecessary files, debug logs, or commented code included.
- [ ] PR has at least 1 peer approval before senior review.
- [ ] I fully understand the code I submitted.
- [ ] No hardcoded configuration or secrets.
- [ ] **Environment Variables** changes? (Leave unchecked if N/A)

# Design & Solution Clarity
- [ ] A design diagram (sequence / flow) is attached or linked.
- [ ] The implementation matches the design.
- [ ] All major flows and decision points are represented in the diagram.
- [ ] Error handling and edge cases are clearly defined in the design.

Design link (diagram / doc):
`<paste link here>`

# Testing
- [ ] Unit tests cover main logic and important edge cases.

> Note: Complex logic must not rely on manual testing only.

# UI evidence (For frontend only)
`<paste screenshots here>`

# Peer Review Checklist
- [ ] I understand the overall solution and flow.
- [ ] The code follows the provided design diagram.
- [ ] No obvious logic or security issues found.
- [ ] Tests are sufficient for the scope of this change.

### CODE REVIEW

**Review by phongvu2812 — APPROVED** [review]

File: dong-service/migrations/029_drop_unique_bank_account_number_constraint.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE dong_schema.user_payment_info
+DROP CONSTRAINT IF EXISTS user_payment_info_bank_name_account_number_key;
+
+ALTER TABLE dong_schema.user_payment_info ADD COLUMN IF NOT EXISTS deleted_at TIMESTAMPTZ;
+
+ON dong_schema.user_payment_info (user_id)
```
Comment (hoangdohuy) [inline dong-service/migrations/029_drop_unique_bank_account_number_constraint.sql]: ? câu migration này là làm gì đây, anh đã run local chưa???

File: frontend/modules/p2p/components/trading-room/buy-amount-section.tsx [hunk 2]
```diff
@@ -176,9 +179,20 @@ export const BuyAmountSection = ({
               {APP_CONFIG.CHAIN_SYMBOL}
             </span>
           </div>
-          <div className="text-muted-foreground mt-2 flex items-center justify-between text-xs">
-            <div>
-              Available: {available.toFormat()} {APP_CONFIG.CHAIN_SYMBOL}
+          <div className="text-muted-foreground mt-2 flex items-start justify-between text-xs">
+            <div className="grid grid-cols-[max-content_auto] gap-x-1 gap-y-0.5">
+              <span>Available:</span>
```
Comment (hoangdohuy) [inline frontend/modules/p2p/components/trading-room/buy-amount-section.tsx:184]: cách dòng ra 1 tí cho đỡ sát

**Review by hoangdohuy — APPROVED** [review]
LGTM
